pages/news/2020-05-10-Mixxx-2-2-4-released.html

```diff
@@ -0,0 +1,41 @@
+title: Mixxx 2.2.4 released
+author: Holzhaus
+date: 2020-10-05 12:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Mixxx 2.2.4 has been released. [Download it here.](http://downloads.mixxx.org/mixxx-2.2.4/)
+
+This release will probably be the last iteration of the 2.2 release, since 2.3 beta is already waiting in the wings.
```

Comment:
```suggestion
This release will probably be the last iteration of the 2.2 release, since 2.3 beta has been released.
```

Reply:
working on the 2.3 beta website PR now

Reply:
I didn't realize we were that close to beta. Reminder to myself: I need to remove the sync master CO from the Roland DJ-505 mapping asap.